hypha/apply/api/v1/tests/test_views.py

```diff
@@ -90,3 +108,249 @@ def test_out_of_order_does_nothing(self):
         self.assertEqual(response_one.status_code, 200, response_one.json())
         self.assertEqual(response_two.status_code, 404, response_two.json())
         self.assertEqual(Activity.objects.count(), 2)
+
+
+@override_settings(ROOT_URLCONF='hypha.apply.urls')
+class TestInvoiceDeliverableViewset(TestCase):
```

Comment:
@sandeepsajan0 can we also add a test with not letting deliverables to be removed once the invoice has been approved.

Staff can not removed deliverables if the invoice is in APPROVED BY STAFF state
Finance 1/Staff can not remove deliverables if the invoice is in APPROVED BY FINANCE 1 state
No one can remove deliverables once the invoice has been approved by finance 2

Basically a few tests for removing deliverables based on user and invoice state.

Reply:
Yes, Sure. That would make it more covered.